docs/references/sdk-metrics.mdx

```diff
@@ -452,3 +459,49 @@ The Schedule-To-Start time of a Workflow Task.
 - Type: Histogram
 - Available in: Core, Go, Java
 - Tags: `namespace`, `task_queue`
+
+### nexus_poll_no_task
+
+A Nexus Worker poll for a Nexus Task timed out, and no Nexus Task is available to pick from the Task Queue.
+
+- Type: Counter
+- Available in: Go, Java
+- Tags: `namespace`, `task_queue`
+
+### nexus_task_schedule_to_start_latency
+
+The Schedule-To-Start time of a Nexus Task in seconds. The schedule time is taken from when the corresponding request
+hit the Frontend service to when the SDK started processing the task.
+
+This time is limited by the `Request-Timeout` header given to the Frontend when handling this request.
+
+- Type: Histogram
+- Available in: Core, Go, Java
+- Tags: `namespace`, `task_queue`, `nexus_service`, `nexus_operation`
+
+### nexus_task_execution_failed
+
+Handling of a Nexus Task resulted in a failed result. Does not take into account unsuccessful Operation results when
+starting an Operation.
```

Comment:
We may want to fix this. Seems somewhat inconsistent.

Reply:
What do we do in Java there?

Reply:
Sorry, what do we want to fix?

Reply:
This is accurate for Java

Reply:
Yeah I could see that , I think that is more consistent with what we do for activities

Reply:
PR to change Go: https://github.com/temporalio/sdk-go/pull/1664

Reply:
I'll fix the comment here.

Reply:
Done.